pages/common/uptime.md

```diff
@@ -0,0 +1,7 @@
+# uptime
+
+> Tell how long the system has been running and other information
+
+- Print current time, uptime, logged users amount and other information
```

Comment:
As it is merged do a new pull-request please. My English is bad (sorry for this), hence I am always doing pull requests to hear everybody's opinions.